### ISSUE TITLE
Menubar Dropdown keyboard navigation

### ISSUE THREAD

**Issue by pmartin35** [salesforce/design-system-react#2901]
Hey!

Currently, if I add something like the code below, the individual dropdowns are unaware of each other through keyboard navigation. If I am focused on an item within the first Dropdown menu and I hit the left/right arrow key, focus should be moved to the second dropdown. Does SLDS provide a wrapper component that handles this sort of keyboard navigation?

```
<nav>
    <SLDSDropdown><Trigger /></SLDSDropdown>
    <SLDSDropdown><Trigger /></SLDSDropdown>
</nav>
```

[Here](https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html) is an example of what I am looking for. Using a keyboard, I can focus on the "About" menu. Using the down arrow opens the menu and focuses the "Overview" item. If I then hit the right arrow key, the next dropdown in the menubar is focused.

**Comment by interactivellama** [comment]
There's no pattern for a `menubar` present in https://www.lightningdesignsystem.com/ 
Global Navigation is the closest, but I believe it uses tabbing. That said, I could see a usage for a Menubar wrapping component that takes Dropdowns.
